readme.md

```diff
@@ -164,7 +164,7 @@ Join us on [GH Discussions][chat]!
 
 ## License
 
-[MIT][] © [Compositor][] and [Vercel][]
+[MIT][] © [Vercel][]
```

Comment:
```suggestion
[MIT][] © Compositor and [Vercel][]
```

I meant remove the link, not remove the text. Sorry.
Copyright can’t just be removed either!